tmt/base.py

```diff
@@ -3900,7 +3902,8 @@ def _stop_running_guests(self, run: Run) -> bool:
     def guests(self, run_ids: tuple[str, ...]) -> bool:
         """ Clean guests of runs """
         self.info('guests', color='blue')
-        root_path = Path(self.opt('workdir-root'))
+        root_path = self.workdir_root
+        self.verbose(f"workdir: {root_path}")
```

Comment:
1. it's not a workdir you're logging, it's a workdir root,
2. `verbose()` and other logging methods can handle label/value or key/value pairs out of the box, taking care of indentation and colon. Together, the logging should look like this: `self.verbose('workdir', root_path)`